src/utils.py

```diff
@@ -486,3 +487,64 @@ def set_test_env(images, images_env="DLC_IMAGES", **kwargs):
 def get_codebuild_project_name():
     # Default value for codebuild project name is "local_test" when run outside of CodeBuild
     return os.getenv("CODEBUILD_BUILD_ID", "local_test").split(":")[0]
+
+def get_safety_ignore_dict(image_uri):
+    """
+    Get a dict of known safety check issue IDs to ignore, if specified in file ../data/ignore_ids_safety_scan.json.
+    :param image_uri:
+    :return: <list> list of safety check IDs to ignore
+    """
+    framework = ("mxnet" if "mxnet" in image_uri else
+                 "pytorch" if "pytorch" in image_uri else
+                 "tensorflow")
+    job_type = ("training" if "training" in image_uri else
+                "inference-eia" if "eia" in image_uri else
+                "inference-neuron" if "neuron" in image_uri else
+                "inference")
+    python_version = "py2" if "py2" in image_uri else "py3"
+
+    IGNORE_SAFETY_IDS = {}
+    with open(f'{os.getenv("ROOT_FOLDER_PATH")}/data/ignore_ids_safety_scan.json') as f:
```

Comment:
Please use `os.path.join()` instead of string formatting to generate file paths.

Reply:
Done